charts/seqr/values.yaml

```diff
@@ -179,6 +174,56 @@ redis:
     enabled: false
   fullnameOverride: "seqr-redis"
 
+postgresql:
+  enabled: false
```

Comment:
wouldn't it be better to enable this by default and just disable it for us in tg-0helm?

Reply:
👍 yes, that works!  I was following the pattern of `redis`, but we can maybe just enable both of them in the open source chart? 

One other option is leaving disabled by here but enabling within the yet to exit `seer-platform` chart, 🤷

Reply:
okay yeah I did not realize redis was off by default. We should def have postgres and redis behave identically in the open source chart, up to you which approach you prefer

Reply:
I think it makes more sense to enable them both here and disable `postgresql` in our chart.